desktop-widgets/templatelayout.cpp

```diff
@@ -319,6 +319,16 @@ static std::vector<const cylinder_t *> cylinderList(const dive *d)
 	return res;
 }
 
+static std::vector<const QString> pictureList(const dive *d)
+{
+	std::vector<const QString> res;
+	res.reserve(d->pictures.nr); //FIXME
```

Comment:
always love seeing FIXME comments... :(

Reply:
According to April 1st, "FIXME" is there as a reminder to remove the "FIXME" comment. (The actual thing to fix I had already fixed but of course forgot the comment).